feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/BottomSheetDaySessionsFragment.kt

```diff
@@ -152,6 +152,13 @@ class BottomSheetDaySessionsFragment : DaggerFragment() {
                 scrollToCurrentSession()
             }
         }
+
+        sessionPagesStore.firstLoadSessions.changed(viewLifecycleOwner) {
+            if (!sessionPagesStore.isFirstLoadScrolled) {
+                scrollToCurrentSession()
+                sessionPagesStore.isFirstLoadScrolled = true
```

Comment:
Can you do it by action? It will break Flux flow 😭

Reply:
That is right...I'm sorry... :bowing_man: 
I change it